### DISCUSSION TITLE
rule: support aws_iam_policy_document_sid_required

### DISCUSSION THREAD

**Discussion by major0** [terraform-linters/tflint-ruleset-aws#641] [Ideas]
Support a check to require a sid in each policy statement.

**Comment by kayman-mk** [comment]
@wata727 What do you think? Seems to be an easy one. 

Edit: in case heredocs are not used

**Reply by wata727** [reply]
It depends on how many people this rule is useful to. If we can't add it as a default enabled rule, a tool like [OPA ruleset](https://github.com/terraform-linters/tflint-ruleset-opa) might be better.
